### DISCUSSION TITLE
3월 8일 회의록

### DISCUSSION THREAD

**Discussion by zangsu** [zangsu/cs-test-service#1] [Announcements]
깃허브 환경 구축을 위해 어떤 과업들이 필요할까요?

**Comment by Moon-GD** [comment]
우선 Wiki에 깃 커밋과 브랜치 네이밍 컨벤션을 함께 맞춰보는 것도 좋을 것 같아요!

**Reply by zangsu** [reply]
네, 지금 바로 구상해 보겠습니다!

**Reply by zangsu** [reply]
### 참여자
장혁수, 문경덕

<br>

### 작업 내용
+ 깃 커밋 컨벤션
+ 브랜치 컨벤션

<br>

->  페어 프로그래밍으로 함께 작성 <a href = "https://github.com/zangsu/cs-test-service/wiki"> Git Wiki / 개발문화 </a>

**Comment by Moon-GD** [comment]
저희 PR 및 코드 작성에 관한 컨벤션도 페어 프로그래밍으로 작성해보면 좋을 것 같아요!

**Reply by Moon-GD** [reply]
코드 작성 관해서는 공통, FE, BE 나누어서 작성해보면 어떨까요?

**Reply by zangsu** [reply]
좋은 생각인 것 같습니다! 바로 구상해 보면 좋을 것 같습니다!!

**Reply by zangsu** [reply]
### 참여자
장혁수, 문경덕

<br>

### 작업 내용
+ PR 컨벤션
+ 코드 컨벤션

<br>

->  페어 프로그래밍으로 함께 작성 <a href = "https://github.com/zangsu/cs-test-service/wiki"> Git Wiki / 개발문화 </a>

**Comment by Moon-GD** [comment]
Git Issue/PR Template도 함께 등록해볼까요?

**Reply by zangsu** [reply]
네, 템플릿 구상해 보면 좋을 것 같습니다!

**Reply by zangsu** [reply]
### 참여자
장혁수, 문경덕

<br>

### 작업 내용
+ PR 템플릿 등록
+ Issue 템플릿 등록

<br>

->  페어 프로그래밍으로 함께 작성
